activerecord/lib/active_record/associations/nullified_owner_attributes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ActiveRecord::Associations
```

Comment:
Nitpick: I wouldn't mix compact and nested class definition styles, but it appears to be that way for `ForeignAssociation`. Still, I would consider changing it to all nested.

Reply:
I also prefer nested class definitions but I added it because of ForeignAssociation.